app/jobs/send_booking_mail_to_passenger_job.rb

```diff
@@ -0,0 +1,7 @@
+class SendBookingMailToPassengerJob < ApplicationJob
+  def perform(booking)
+    booking.passengers.each do |passenger|
+      BookingMailer.flight_booked(booking, passenger).deliver_later
+    end
+  end
+end
```

Comment:
The job enqueue the confirmation emails to the passengers

Reply:
I'd suggest calling this `SendBookingMailToPassengersJob`, with passengers in plural. Or `SendBookingMails`